barrier_1:
  joint_1_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  gear_to_barrier_1_position_controller:
    type: effort_controllers/JointEffortController
    joint: gear_to_barrier_1

barrier_2:
  joint_2_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  gear_to_barrier_2_position_controller:
    type: effort_controllers/JointEffortController
    joint: gear_to_barrier_2
    
barrier_1/gazebo_ros_control/pid_gains:
  gear_to_barrier_1: {p: 100.0, i: 0.01, d: 10.0}

barrier_2/gazebo_ros_control/pid_gains:
  gear_to_barrier_2: {p: 100.0, i: 0.01, d: 10.0}

traffic_lights_1:
  light_1_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  
  base_to_light_1_position_controller:
    type: effort_controllers/JointEffortController
    joint: base_to_light_green_1

  light_2_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  
  base_to_light_2_position_controller:
    type: effort_controllers/JointEffortController
    joint: base_to_light_orange_1

  light_3_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  base_to_light_3_position_controller:
    type: effort_controllers/JointEffortController
    joint: base_to_light_red_1

traffic_lights_1/gazebo_ros_control/pid_gains:
  base_to_light_green_1: {p: 100.0, i: 0.01, d: 10.0}
  base_to_light_orange: {p: 100.0, i: 0.01, d: 10.0}
  base_to_light_red: {p: 100.0, i: 0.01, d: 10.0}



